### PR TITLE
:hammer: change of cmd clear

### PR DESCRIPTION
**Pull request by foltone**
Add one option "Select a specific user".
Change of location : info to moderation.

### CODE REVIEW

File: src/slscommands/moderation/clear.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    if (target) {
+      let i = 0;
+      const filterCibleMessages = [];
+      (await messageToDelet).filter((msg) => {
+        if (msg.author.id == target.id && number > i) {
+          filterCibleMessages.push(msg); i++;
+        }
+      });
+      await interaction.channel.bulkDelete(filterCibleMessages, true).then((messages) => {
+        interaction.channel.send(`${messages.size} message from ${target}, has been deleted!`);
```
Comment (a9ex) [inline src/slscommands/moderation/clear.js:37]:
```suggestion
        interaction.reply(`${messages.size} message from ${target}, has been deleted!`);
```

File: src/slscommands/moderation/clear.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      });
+    } else {
+      await interaction.channel.bulkDelete(number, true).then((messages) => {
+        interaction.channel.send(`${messages.size} messages have been successfully deleted!`);
```
Comment (a9ex) [inline src/slscommands/moderation/clear.js:41]:
```suggestion
        interaction.reply(`${messages.size} messages have been successfully deleted!`);
```

File: src/slscommands/moderation/clear.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  name: 'clear',
+  description: 'To delete some messages in channels.',
+  permissions: ['MANAGE_MESSAGES'],
+  type: 1,
+  options: [
+    {
+      name: 'number',
+      description: 'Give the number of message deleted.',
+      required: true,
+      type: 'NUMBER',
```
Comment (a9ex) [inline src/slscommands/moderation/clear.js:11]: You pass a `min_value` & `max_value`  :
image
```suggestion
      type: 'NUMBER',
      min_value: 1,
      max_value: 99
```

**Review by a9ex — CHANGES_REQUESTED** [review]

File: src/slscommands/moderation/clear.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+    if (target) {
+      let i = 0;
+      const filterCibleMessages = [];
+      (await messageToDelet).filter((msg) => {
+        if (msg.author.id == target.id && number > i) {
+          filterCibleMessages.push(msg); i++;
+        }
+      });
+      await interaction.channel.bulkDelete(filterCibleMessages, true).then((messages) => {
```
Comment (a9ex) [inline src/slscommands/moderation/clear.js:36]: there's no need to use `await` if you use `.then()`

**Review by a9ex — CHANGES_REQUESTED** [review]

File: src/slscommands/moderation/clear.js (hunk 2 above)
Comment (foltone) [reply src/slscommands/moderation/clear.js:41]: if I do that, the message.reply will be deleted

**Comment by foltone** [comment]
I made the command waiting message delete itself

**Review by a9ex — APPROVED** [review]

**Pull request merged**